packages/composite-checkout/package.json

```diff
@@ -48,7 +48,7 @@
 	},
 	"devDependencies": {
 		"@automattic/calypso-build": "^9.0.0",
-		"@automattic/calypso-polyfills": "^1.0.0",
+		"@automattic/calypso-polyfills": "^2.0.0",
```

Comment:
I wonder if this is even needed in this package? 🤔  I don't remember why it was added; it may have been for a test that's been removed.

Reply:
You're right, it is not needed.

I added it in #45872 because it was needed in the demo site, but I removed that requirement when I migrated it to Storybook (#52194), and forgot to delete the dependency.